tissue_bridges/README.md

```diff
@@ -0,0 +1,86 @@
+
+1. **_manual_**: manual measurement of tissue bridges on manually segmented (GT) intramedullary lesions -- values provided by collaborators
+2. **_semi-automatic_**: automatic measurement of the tissue bridges using the proposed method on manually segmented (GT) intramedullary lesions (located under `derivatives/labels`)
+3. **_automatic_**: automatic measurement of tissue bridges using SCIsegV2 predictions
+
+## 0. Generate QC
+
+Generate QC to compare ground truth vs predicted lesions.
+
+### Ground truth
+```bash
+cd ~/data/data.neuro.polymtl.ca/sci-zurich
+for sub in sub-zh1[0-9][0-9];do sct_qc -i ${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w.nii.gz -d derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_lesion-manual.nii.gz -s derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_seg-manual.nii.gz -p sct_deepseg_lesion -plane sagittal -qc-subject $sub -qc ~/data/results/sci-zurich/sci-zurich_2024-06-18_manualGT/qc;done
+```
+
+### SCIsegV2 predictions
+
+```bash
+cd ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0
+for file in *sag*lesion.nii.gz;do sub=$(echo $file | grep -o "sub-zh1..");sct_qc -i ~/data/data.neuro.polymtl.ca/sci-zurich/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w.nii.gz -d $file -s ${file/lesion/seg} -p sct_deepseg_lesion -plane sagittal -qc-subject $sub -qc ./qc;done
+```
+
+## 1. Compute tissue bridges
+
+### Ground truth - "semi-automatic" method
+
+Compute tissue bridges using `sct_analyze_lesion` from GT lesion and spinal cord segmentations (located under 
+`sci-zurich/derivatives/labels`). The GT segmentations are organized according to the BIDS, so we can use 
+`sct_run_batch` wrapper script to process all subjects.
+
+Example usage:
+
+```bash
+sct_run_batch -config config-01a_compute_tissue_bridges_from_GT.json
+```
+
+### SCIsegV2 predictions - "automatic" method
+
+Compute tissue bridges using sct_analyze_lesion based from lesion and spinal cord segmentations obtained using `SCIsegV2`.
+The predictions are in a single folder (i.e., non BIDS-compliant), so we cannot use `sct_run_batch` wrapper script.
+
+Example usage:
+
+```bash
+cd ~/code/sci-zurich/tissue_bridges
+
+./01b_compute_tissue_bridges_from_predictions.sh ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0
+```
+
+> Note:
+> For both GT and predictions, `sct_analyze_lesion` outputs one XLS file per subject. --> we need to merge them into a 
+> single file, see the next step.
+
+
+## 2. Read and merge XLS files
+
+Read XLS files (generated by `sct_analyze_lesion`; one file per subject) located under `/results` and merge them into a
+single file.
+
+Example usage:
+
+```bash
+cd ~/code/sci-zurich/tissue_bridges
+
+python 02_read_xls_files.py
+-dir ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/results
+-o ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/stats
```

Comment:
```suggestion
python 02_read_xls_files.py \
-dir ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/results \
-o ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/stats \
```

I thought multi-line python commands should always have a trailing `\` ? (if not, then please ignore the suggestion)

Reply:
good catch! agree, thanks!